transmart-core-db/grails-app/domain/org/transmartproject/db/ontology/TableAccess.groovy

```diff
@@ -163,12 +166,85 @@ class TableAccess extends AbstractQuerySpecifyingType implements
         getDescendants(true, showHidden, showSynonyms)
     }
 
+    @Override
+    List<OntologyTerm> getHDforAllDescendants() {
+        getHDDescendants(true, false, false, false)
+    }
+
+    List<String> getAllDescendantsForFacets() {
+        getDescendantsFullName(true, false, false, false)
+    }
+
+    private List<OntologyTerm> getHDDescendants(boolean allDescendants,
```

Comment:
Maybe add one parameter to `getDescendantsFullName` instead of duplicating the function?